src/platforms/python/common/configuration/integrations/loguru.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Loguru
+description: Learn about using Sentry with loguru.
+sidebar_order: 80
+---
+
+The [Loguru](https://github.com/Delgan/loguru#readme) integration lets you capture log messages and send them to Sentry.
+
+Most of the work is handled by the [`logging`](/platforms/python/guides/logging) integration, so most of the examples there will work with loguru.
+
+## Install
+
+Install `sentry-sdk` from PyPI with the `loguru` extra.
+
+```bash
+pip install --upgrade 'sentry-sdk[loguru]'
+```
+
+## Configure
+
+```python
+import sentry_sdk
+from sentry_sdk.integrations.loguru import LoguruIntegration
+
+sentry_sdk.init(
+    dsn="___PUBLIC_DSN___",
+    integrations=[
+        LoguruIntegration(),
+    ],
+)
+```
+
+## Behavior
+
+By default log records with a level of `INFO` or higher will be added as breadcrumbs to Sentry events. For log records with a level of `ERROR` or higher a Sentry issue will be created:
+
+```python
+from loguru import logger
+
+logger.debug("I am ignored")
+logger.info("I am a breadcrumb")
+logger.error("I am an event", extra=dict(bar=43))
+logger.exception("An exception happened")
+```
+
+- There will be an error event with the message `"I am an event"`.
+- `"I am a breadcrumb"` will be attached as a breadcrumb to that event.
+- `bar` will end up in the `extra` attributes of that event.
+- `"An exception happened"` will send the current exception from `sys.exc_info()` with the stack trace to the Sentry. If there's no exception, the current stack will be attached.
+- The debug message `"I am ignored"` will not be captured by Sentry. To capture it, you need to lower `level` to `DEBUG` in `LoguruIntegration`.
+
+### Ignoring a logger
+
+Sometimes a logger can be noisy. You can ignore that logger by calling `ignore_logger`:
+
+```python
```

Comment:
This code snippet didn't make sense to me. At first I thought it was showing two ways of doing the same thing (one with `logging` and one with `loguru`, but I don't think that's the case.

Is it showing code that should be in two different files? If so, this should be in 2 separate code snippet boxes.

Maybe this would make sense to loguru users, and if so, feel free to ignore!

Reply:
Yes, the example is incomplete because of https://github.com/getsentry/sentry-docs/pull/6617/commits/c0e507a7c70dc4c54540b265679b816c023abfc7#r1193011580.